go-controller/pkg/ovn/secondary_layer3_network_controller_test.go

```diff
@@ -239,6 +246,13 @@ var _ = Describe("OVN Multi-Homed pod operations", func() {
 			icClusterWithDisableSNATTestConfiguration(),
 			config.GatewayModeShared,
 		),
+		Entry("pod on a user defined primary network on an IC cluster with EgressFirewall enabled",
+			dummyPrimaryLayer3UserDefinedNetwork("192.168.0.0/16", "192.168.1.0/24"),
+			icClusterTestConfiguration(func(config *testConfiguration) {
+				config.configToOverride.EnableEgressFirewall = true
```

Comment:
++ I like this better, I wanted to say can you do it for the other It entries as well but that's a cleanup and I cannot ask you to do it cause its not related to your PR. but if you ever get time, would love this change in favor or removing the other existing utils.

Reply:
yes @tssurya , updated  existing utils.